### PR TITLE
Specify other alternatives for the Status object

### PR DESCRIPTION
**Pull request by WalkQuackBack**
## Changes

Mentioned how Character Physics Controllers can be used as a way to add custom states

https://devforum.roblox.com/t/releasing-character-physics-controllers/2623426

## Checks

By submitting your pull request for review, you agree to the following:

- [x] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [x] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [x] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

File: content/en-us/reference/engine/classes/Status.yaml [hunk 1]
```diff
@@ -19,8 +19,7 @@ deprecation_message: |
   work.
 
   Developers looking to implement custom `Class.Humanoid` statuses should use
-  `Class.CollectionService` which provides a more flexible and robust system for
-  tagging objects.
+  [Character Physics Controllers](https://devforum.roblox.com/t/releasing-character-physics-controllers/2623426), as they are written in Luau and are easily extendable.
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/Status.yaml:22]:
```suggestion
  [Character Physics Controllers](../../../physics/character-controllers.md), as they are written in Luau and are easily extendable.
```

Comment (IgnisRBX) [reply content/en-us/reference/engine/classes/Status.yaml:22]: Changing link to the guide on this, which is the same feature as the forum post:
https://create.roblox.com/docs/physics/character-controllers

Comment (WalkQuackBack) [reply content/en-us/reference/engine/classes/Status.yaml:22]: I couldn't find that page while doing research, I'll change out the DevForum link for that!

**Comment by WalkQuackBack** [comment]
I'm going to re-sync this pull request as there's been changes since then, there should be no conflicts

**Review by IgnisRBX — APPROVED** [review]

**Pull request merged**